### PR TITLE
Most recently used tab selector

### PR DESCRIPTION
**Pull request by alx741**
Default mapping after vimperator

`<C-^>` ===  `<C-6>`: Select the most recently used tab

Mimics the behavior of vimperator

### CODE REVIEW

**Comment by alx741** [comment]
Alright, the merge conflicts are now solved

**Comment by alx741** [comment]
Solves #164

**Comment by ueokande** [comment]
@alx741 That's nice enhanvecent, but your changes have a problem: It does not work for tabs switched by a mouse or key bindings in Firefox.

**Comment by alx741** [comment]
mmh! didn't thought about that, will see if there's a way to work that out

**Comment by alx741** [comment]
@ueokande Now it works as expected whenever the active tab changes, whether it is using vixen's commands/mappings, the mouse, or firefox's owns key bindings

File: README.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ The default mappings are as follows:
 - <kbd>u</kbd>: reopen close tab
 - <kbd>K</kbd>, <kbd>J</kbd>: select prev or next tab
 - <kbd>g0</kbd>, <kbd>g$</kbd>: select first or last tab
+- <kbd>Ctrl</kbd>+<kbd>^</kbd>: select previous selected tab
```
Comment (ueokande) [inline README.md]: It differs from actual keys. Use `kbd>Ctrl</kbd>+<kbd>6</kbd>`.

Comment (ueokande) [inline README.md]: It differs from actual keys. Use `kbd>Ctrl</kbd>+<kbd>6</kbd>`.

Comment (ueokande) [inline README.md]: It differs from actual keys. Use `kbd>Ctrl</kbd>+<kbd>6</kbd>`.

**Pull request merged**